sdk/ml/azure-ai-ml/azure/ai/ml/_schema/schedule/trigger.py

```diff
@@ -19,7 +19,7 @@
 class TriggerSchema(metaclass=PatchedSchemaMeta):
     start_time = UnionField([fields.DateTime(), DateTimeStr()])
     end_time = UnionField([fields.DateTime(), DateTimeStr()])
-    time_zone = fields.Str(validate=validate.OneOf([o.value for o in TimeZone]))
+    time_zone = fields.Str()
```

Comment:
why skip the check?

Reply:
Our backend service has support IANA timezone in trigger service, and this field may be changed afterwards, so we are going to keep the enum class as it is now and just removing the limitation for blocking user submission.